Analysis/config/2016/sources.cfg

```diff
@@ -26,8 +26,9 @@ xs_cfg: hh-bbtautau/Instruments/config/cross_section.cfg
 plot_page_opt: internal_plot
 use_svFit: true
 use_kinFit: true
-use_IterativeFit: true
 allow_calc_svFit: false
+use_IterativeFit: true
+apply_JES: true
```

Comment:
for the moment we want to save both versions of btag SF shifts with JES for btag applied and not -> don't create a param the config.